systemd-patches/9504-core-skip-restart-when-a-JOB_STOP-job-is-pending.patch

```diff
@@ -0,0 +1,30 @@
+From 9b2f19a583b98e47ffa1f2bdf930eced4f537406 Mon Sep 17 00:00:00 2001
+From: Kyle Sessions <kssessio@amazon.com>
+Date: Sat, 4 May 2024 00:26:08 +0000
+Subject: [PATCH] core: skip automatic restart when a JOB_STOP job is pending
+
+---
+ src/core/service.c | 7 +------
+ 1 file changed, 1 insertion(+), 6 deletions(-)
```

Comment:
Since this isn't your original patch, please use the same header that you get from `git format-patch` for https://github.com/systemd/systemd/commit/c972880640ee19e89ce9265d8eae1b3aae190332 and add these lines to the end:
```
[kssessio: backport to v219]
Signed-off-by: Kyle Sessions <kssessio@amazon.com>
```

This is essentially the process described here:
https://www.kernel.org/doc/html/next/process/backporting.html#submitting-backports-to-stable

Goal is to give credit to the originator of the fix while taking credit (blame) for the backport 😅

Reply:
Thanks for the link and for catching that. 

I have updated it! Let me know if you see any issues with it!